airbyte-commons-worker/src/main/java/io/airbyte/workers/helper/CatalogClientConverters.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.workers.helper;
+
+import io.airbyte.commons.enums.Enums;
+import io.airbyte.commons.text.Names;
+import io.airbyte.protocol.models.AirbyteStream;
+import java.util.stream.Collectors;
+
+/**
+ * Utilities to convert Catalog protocol to Catalog API client.
+ */
+public class CatalogClientConverters {
```

Comment:
@mfsiega-airbyte curious if you have thoughts here on adding more 'catalog conversion' logic to a new module. These conversion functions are new in that they're converting to the _client_ (ie, 'api.client.model.generated') types as opposed to the _api_ types that the existing CatalogConverter.java handles. 

I think the 'airbyte-commons-worker' module probably makes sense for client-specific conversions (since the worker here is after all a client of the API) but I feel like I want to raise a discussion here to make sure we aren't fragmenting messy catalog conversion logic all over the place if we can avoid it.  Thoughts?

Reply:
Want to add that I can't add this logic into CatalogConverters because it is defined in airbyte-common-server, which has already use airbyte-common-workers as its dependency; introducing this will cause the dependency loop error.

I was also thinking about moving both code into a common library so airbyte-common-server and airbyte-common-worker can both depend on it, but not sure where to put it; plus seems server side code will never be used by worker and client side code will never be used by server.

Reply:
i feel this might be out of scope for this PR - I'll create a separate ticket to track this and we probably should discuss refactoring goal/plan in a refining meeting

Reply:
All of the above makes sense to me, thanks for creating that followup ticket @xiaohansong !